core/src/test/scala/org/broadinstitute/dsde/rawls/monitor/AvroUpsertMonitorSpec.scala

```diff
@@ -622,6 +622,71 @@ class AvroUpsertMonitorSpec(_system: ActorSystem)
       }
   }
 
```

Comment:
the below test is copied from the pre-existing "bubble up useful error message if upserts result in partial failure" test just above it, changed only to ensure that all entities fail to import and to expect a slightly different error message in the end.